packages/yoroi-extension/app/containers/wallet/WalletReceivePage.js

```diff
@@ -119,6 +120,7 @@ class WalletReceivePage extends Component<AllProps> {
     };
 
     const isHwWallet = publicDeriver.type !== 'mnemonic';
+    const isAddressBook = this.props.stores.router.location.pathname.startsWith(`${ROUTES.WALLETS.RECEIVE.ROOT}/addressBook/all`)
```

Comment:
Please define the address book flag by the code already used on this same page:
```
addressTypeStore.meta.name.group === AddressGroupTypes.addressBook
```

You can then also replace multiple usages of this code expression by the use of your new flag `isAddressBook`

Reply:
Thank you! I didn't notice that. Will do shortly.

Reply:
Fixed.